android-pay/src/main/java/com/stripe/wrap/pay/utils/PaymentUtils.java

```diff
@@ -254,14 +255,14 @@ public static CartError searchLineItemForErrors(LineItem lineItem) {
 
     /**
      * Converts an integer price in the lowest currency denomination to a Google string value.
-     * The currency is assumed to be from {@link Locale#getDefault()}.
+     * The currency is assumed to be from the current {@link AndroidPayConfiguration} singleton.
      *
      * @param price the price in the lowest available currency denomination
      * @return a String that can be used as an Android Pay price string
      */
     @NonNull
     public static String getPriceString(long price) {
-        return getPriceString(price, Currency.getInstance(Locale.getDefault()));
```

Comment:
Moved away from using `Locale.getDefault()` as default currency, and instead use Dollars.